cbits/bitvec_simd.c

```diff
@@ -0,0 +1,65 @@
+#include <inttypes.h>
+#include <stddef.h>
+
+void omp_com(uint8_t *dest, uint8_t *src, size_t len) {
```

Comment:
I suspect that we need
```suggestion
void _hs_bitvec_omp_com(uint8_t *dest, uint8_t *src, size_t len) {
```
and similar below.

Reply:
You mean to avoid naming conflicts?

Reply:
Yes. At least this is what `bytestring` and `text` do.